receiver/otlpreceiver/config.md

```diff
@@ -0,0 +1,91 @@
+# "otlp" Receiver Reference
+
+Config defines configuration for OTLP receiver.
+
+
+### Config
+
+| Name | Type | Default | Docs |
+| ---- | ---- | ------- | ---- |
+| protocols |[otlpreceiver-Protocols](#otlpreceiver-Protocols)| <no value> | Protocols is the configuration for the supported protocols, currently gRPC and HTTP (Proto and JSON).  |
+
+### otlpreceiver-Protocols
```

Comment:
Use like the package reference `otlpreceiver.Protocols`?

Reply:
Yeah, I wanted this as well, but I wasn't able to get anchor scrolling to work using a dot. I also tried `&period;` but that didn't work either: the link shows up, but browsers won't scroll to the table that has the definition for that type.

Here's a rendered document:

https://github.com/pmcollins/opentelemetry-collector/blob/schema-api/receiver/otlpreceiver/config.md

If you click on one of the links in the 'type' column, the browser scrolls to the table for that type.

I also thought about using an underscore, but the dash at least allows for the type to be broken into two lines when there's not enough room. Not sure -- maybe there's another symbol that would work better?